drizzle-orm/src/mysql-core/columns/varbinary.ts

```diff
@@ -15,8 +15,8 @@ export interface MySqlVarBinaryHKT extends ColumnHKTBase {
 
 export type MySqlVarBinaryBuilderInitial<TName extends string> = MySqlVarBinaryBuilder<{
 	name: TName;
-	data: number;
-	driverParam: number | string;
+	data: Buffer;
+	driverParam: Buffer | string;
```

Comment:
If you're not doing any data mapping, the `driverParam` should be the same type as `data`.